docs/pages/index.mdx

```diff
@@ -7,7 +7,7 @@ import { HomePage } from "vocs/components";
 
 <HomePage.Root>
   <h1 className="vocs_HomePage_title">frames.js</h1>
-  <HomePage.InstallPackage name="frames.js" type="install" />
+  <HomePage.InstallPackage name="frames" type="init" />
```

Comment:
i think we should leave the install as the initial call to action since we have create-frames in the quickstart section

Reply:
Done